### PR TITLE
More verbose status docs

### PR DESCRIPTION
**Pull request by SabaPing**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Make the status document more specific and add reference links.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-cluster-info.md [hunk 1]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
 - Git 哈希值 (Git Hash)：实例二进制对应的 Git 哈希值
 - 部署路径 (Deployment Directory)：实例二进制文件所在目录路径
 
-实例的运行状态有：
+### 实例的运行状态 (Status)
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:38]:
```suggestion
### 实例运行状态 (Status)
```

File: dashboard/dashboard-cluster-info.md [hunk 2]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
+实例运行状态有以下几个可能值：
 
 - 在线 (Up)：实例正常运行。
 - 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
 - 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
-- 下线中 (Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Leaving)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+
+  > **注意：**
+  > TiDB Dashboard 显示的 `Leaving` 就是 PD API 返回的 `offline`，同时也是 TiUP 显示的 `Pending Offline`，三者含义相同。
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:48]:
```suggestion
> **注意：**
>
> TiDB Dashboard 显示的 `Leaving`，PD API 返回的 `Offline`，TiUP 显示的 `Pending Offline`，三者含义相同。
```

File: dashboard/dashboard-cluster-info.md [hunk 3]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
+  
 - 未知 (Unknown)：未知的实例运行状态。
 
+实例运行状态来自于 PD 的调度信息。更详细的描述请参考 [《TiDB 数据库的调度》中的“信息收集”小结](https://docs.pingcap.com/zh/tidb/stable/tidb-scheduling#%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86)。
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:52]:
```suggestion
实例运行状态来自于 PD 的调度信息。更详细的描述请参考 [TiDB 数据库的调度 -- 信息收集](/tidb-scheduling.md#信息收集)。
```

File: tiup/tiup-command-status.md [hunk 4]
```diff
@@ -36,4 +36,20 @@ tiup status [flags]
...
+- 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
+- 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Pending Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+
+  > **注意：**
+  > TiUP 显示的 `Pending Offline` 就是 PD API 返回的 `offline`，同时也是 TiDB Dashboard 显示的 `Leaving`，三者含义相同。
+  
+- 未知 (Unknown)：未知的实例运行状态。
+
+组件运行状态来自于 PD 的调度信息。更详细的描述请参考 [《TiDB 数据库的调度》中的“信息收集”小结](https://docs.pingcap.com/zh/tidb/stable/tidb-scheduling#%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86)。
```
Comment (shichun-0415) [inline tiup/tiup-command-status.md:53]:
```suggestion
组件运行状态来自于 PD 的调度信息。更详细的描述请参考 [TiDB 数据库的调度 -- 信息收集](/tidb-scheduling.md#信息收集)。
```

File: tiup/tiup-component-cluster-display.md [hunk 5]
```diff
@@ -80,4 +80,20 @@ tiup cluster display <cluster-name> [flags]
...
+- 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
+- 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Pending Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+  
+  > **注意：**
+  > TiUP 显示的 `Pending Offline` 就是 PD API 返回的 `offline`，同时也是 TiDB Dashboard 显示的 `Leaving`，三者含义相同。
+
+- 未知 (Unknown)：未知的实例运行状态。
+
+组件运行状态来自于 PD 的调度信息。更详细的描述请参考 [《TiDB 数据库的调度》中的“信息收集”小结](https://docs.pingcap.com/zh/tidb/stable/tidb-scheduling#%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86)。
```
Comment (shichun-0415) [inline tiup/tiup-component-cluster-display.md:97]:
```suggestion
组件运行状态来自于 PD 的调度信息。更详细的描述请参考 [TiDB 数据库的调度 -- 信息收集](/tidb-scheduling.md#信息收集)。
```

File: tiup/tiup-component-cluster-display.md [hunk 6]
```diff
@@ -80,4 +80,20 @@ tiup cluster display <cluster-name> [flags]
     - Data Dir：服务的数据目录，`-` 表示没有数据目录
     - Deploy Dir：服务的部署目录
 
+### 服务当前的状态 (Status)
```
Comment (shichun-0415) [inline tiup/tiup-component-cluster-display.md:83]:
```suggestion
### 节点服务的状态 (Status)
```

File: tiup/tiup-component-cluster-display.md [hunk 7]
```diff
@@ -80,4 +80,20 @@ tiup cluster display <cluster-name> [flags]
...
+
+服务当前状态有以下几个可能值：
```
Comment (shichun-0415) [inline tiup/tiup-component-cluster-display.md:85]:
```suggestion
节点服务可能处于如下任一状态：
```

**Review by lilyjazz — APPROVED** [review]

File: tiup/tiup-component-cluster-display.md [hunk 8]
```diff
@@ -80,4 +80,20 @@ tiup cluster display <cluster-name> [flags]
...
+
+- 在线 (Up)：实例正常运行。
+- 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
+- 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Pending Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+  
+  > **注意：**
+  > TiUP 显示的 `Pending Offline` 就是 PD API 返回的 `offline`，同时也是 TiDB Dashboard 显示的 `Leaving`，三者含义相同。
```
Comment (shichun-0415) [inline tiup/tiup-component-cluster-display.md:93]:
```suggestion
  > **注意：**
  >
  > TiUP 显示的 `Pending Offline`，PD API 返回的 `Offline`，TiDB Dashboard 显示的 `Leaving`，三者含义相同。
```

File: tiup/tiup-command-status.md [hunk 9]
```diff
@@ -36,4 +36,20 @@ tiup status [flags]
...
+
+组件运行状态有以下几个可能值：
+
+- 在线 (Up)：实例正常运行。
+- 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
+- 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Pending Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+
+  > **注意：**
+  > TiUP 显示的 `Pending Offline` 就是 PD API 返回的 `offline`，同时也是 TiDB Dashboard 显示的 `Leaving`，三者含义相同。
```
Comment (shichun-0415) [inline tiup/tiup-command-status.md:49]:
```suggestion
  > **注意：**
  >
  > TiUP 显示的 `Pending Offline`，PD API 返回的 `Offline`，TiDB Dashboard 显示的 `Leaving`，三者含义相同。
```

File: dashboard/dashboard-cluster-info.md [hunk 10]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
 - Git 哈希值 (Git Hash)：实例二进制对应的 Git 哈希值
 - 部署路径 (Deployment Directory)：实例二进制文件所在目录路径
 
-实例的运行状态有：
+### 实例的运行状态 (Status)
+
+实例运行状态有以下几个可能值：
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:40]:
```suggestion
实例可能处于如下任一运行状态：
```

File: tiup/tiup-command-status.md [hunk 11]
```diff
@@ -36,4 +36,20 @@ tiup status [flags]
 - Binary: 二进制文件路径
 - Args: 启动参数
 
-[<< 返回上一页 - TiUP 命令清单](/tiup/tiup-reference.md#命令清单)
\ No newline at end of file
+### 组件运行状态 (Status)
+
+组件运行状态有以下几个可能值：
```
Comment (shichun-0415) [inline tiup/tiup-command-status.md:41]:
```suggestion
组件可能处于如下任一运行状态：
```

File: dashboard/dashboard-cluster-info.md [hunk 12]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
 - 在线 (Up)：实例正常运行。
 - 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
 - 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
-- 下线中 (Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Leaving)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+
+  > **注意：**
+  > TiDB Dashboard 显示的 `Leaving` 就是 PD API 返回的 `offline`，同时也是 TiUP 显示的 `Pending Offline`，三者含义相同。
+  
 - 未知 (Unknown)：未知的实例运行状态。
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:50]:
```suggestion
- 未知 (Unknown)：未知的实例运行状态。

> **注意：**
>
> TiDB Dashboard 显示的 `Leaving`，PD API 返回的 `Offline`，TiUP 显示的 `Pending Offline`，三者含义相同。
```

File: dashboard/dashboard-cluster-info.md [hunk 13]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
 
 - 在线 (Up)：实例正常运行。
 - 离线 (Down) 或无法访问 (Unreachable)：实例未启动或对应主机存在网络问题。
 - 已缩容下线 (Tombstone)：实例上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 实例存在该状态。
-- 下线中 (Offline)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+- 下线中 (Leaving)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+
+  > **注意：**
+  > TiDB Dashboard 显示的 `Leaving` 就是 PD API 返回的 `offline`，同时也是 TiUP 显示的 `Pending Offline`，三者含义相同。
+
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:49]:
```suggestion
```

File: dashboard/dashboard-cluster-info.md [hunk 14]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
+  > **注意：**
+  > TiDB Dashboard 显示的 `Leaving` 就是 PD API 返回的 `offline`，同时也是 TiUP 显示的 `Pending Offline`，三者含义相同。
+  
 - 未知 (Unknown)：未知的实例运行状态。
 
+实例运行状态来自于 PD 的调度信息。更详细的描述请参考 [《TiDB 数据库的调度》中的“信息收集”小结](https://docs.pingcap.com/zh/tidb/stable/tidb-scheduling#%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86)。
+
 > **注意：**
 >
 > 表格中部分列仅在实例处于在线状态时能显示。
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:56]:
```suggestion
```

File: dashboard/dashboard-cluster-info.md [hunk 15]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
+- 下线中 (Leaving)：实例上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 实例存在该状态。
+
+  > **注意：**
+  > TiDB Dashboard 显示的 `Leaving` 就是 PD API 返回的 `offline`，同时也是 TiUP 显示的 `Pending Offline`，三者含义相同。
+  
 - 未知 (Unknown)：未知的实例运行状态。
 
+实例运行状态来自于 PD 的调度信息。更详细的描述请参考 [《TiDB 数据库的调度》中的“信息收集”小结](https://docs.pingcap.com/zh/tidb/stable/tidb-scheduling#%E4%BF%A1%E6%81%AF%E6%94%B6%E9%9B%86)。
+
 > **注意：**
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:54]:
```suggestion
```

File: dashboard/dashboard-cluster-info.md [hunk 16]
```diff
@@ -35,14 +35,22 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-cluster-info/']
...
 >
```
Comment (shichun-0415) [inline dashboard/dashboard-cluster-info.md:55]:
```suggestion
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiup/tiup-component-cluster-display.md [hunk 17]
```diff
@@ -80,4 +80,20 @@ tiup cluster display <cluster-name> [flags]
...
+
+- 在线 (Up)：节点服务正常运行。
+- 离线 (Down) 或无法访问 (Unreachable)：节点服务未启动或对应主机存在网络问题。
+- 已缩容下线 (Tombstone)：节点服务上的数据已被完整迁出并缩容完毕。仅 TiKV 或 TiFlash 存在该状态。
+- 下线中 (Pending Offline)：节点服务上的数据正在被迁出并缩容。仅 TiKV 或 TiFlash 存在该状态。
+- 未知 (Unknown)：未知的节点服务运行状态。
+
+> **注意：**
+>
+> TiUP 显示的 `Pending Offline`，PD API 返回的 `Offline`，TiDB Dashboard 显示的 `Leaving`，三者含义相同。
```
Comment (Oreoxmt) [inline tiup/tiup-component-cluster-display.md:95]:
```suggestion
> TiUP 显示的 `Pending Offline`、PD API 返回的 `Offline` 以及 TiDB Dashboard 显示的 `Leaving` 这三个状态的含义相同。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**